pkg/cloud/provider/provider_test.go

```diff
@@ -42,3 +46,130 @@ func TestParseLocalDiskID(t *testing.T) {
 		})
 	}
 }
+
+func Test_getClusterProperties(t *testing.T) {
```

Comment:
Added a test for `getClusterProperties`, even though I was unable to repro the issue at hand locally.